src/components/Menu.tsx

```diff
@@ -1,39 +1,35 @@
 import { Bug, Cog, Files, FlaskConical, GitFork, Monitor, Puzzle, Search, User } from 'lucide-react'
+import type { Icon } from 'lucide-react'
 
 export function Menu() {
   return (
     <div className="flex flex-col justify-between">
       <div className="flex flex-col">
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-[#E0DEF2]">
-          <Files strokeWidth={1.5} size={28} color="#E0DEF2" />
-        </button>
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <Search strokeWidth={1.5} size={28} color="#8F8CA8" />
-        </button>
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <GitFork strokeWidth={1.5} size={28} color="#8F8CA8" />
-        </button>
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <Bug strokeWidth={1.5} size={28} color="#8F8CA8" />
-        </button>
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <Puzzle strokeWidth={1.5} size={28} color="#8F8CA8" />
-        </button>
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <Monitor strokeWidth={1.5} size={28} color="#8F8CA8" />
-        </button>
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <FlaskConical strokeWidth={1.5} size={28} color="#8F8CA8" />  
-        </button>
+        <Button Icon={Files} isActive />
+        <Button Icon={Search} />
+        <Button Icon={GitFork} />
+        <Button Icon={Bug} />
+        <Button Icon={Puzzle} />
+        <Button Icon={Monitor} />
+        <Button Icon={FlaskConical} />
       </div>
       <div className="flex flex-col items-center">
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <User strokeWidth={1.5} size={28} color="#8F8CA8" />
-        </button>
-        <button type="button" className="h-12 flex justify-center items-center border-l-2 border-transparent">
-          <Cog strokeWidth={1.5} size={28} color="#8F8CA8" />
-        </button>
+        <Button Icon={User} />
+        <Button Icon={Cog} />
       </div>
     </div>
   )
+}
+
+interface ButtonProps {
+  Icon: Icon
+  isActive?: boolean
+}
+
+function Button({ Icon, isActive = false }: ButtonProps) {
+  return (
+    <button type="button" className={"h-12 flex justify-center items-center border-l-2" + " " + (isActive ? "border-[#E0DEF2]" : "border-transparent")}>
```

Comment:
Today these buttons are not being used for anything so maybe we could use them as `div` instead of `buttons` so the screen reader will not read them as clickable things. What do you think?

Reply:
@mateusabelli 
Are you gonna make the changes anytime soon ?

I will add dropdown feature to user icon, and it will need to change back this element to button, because I will use radix dropdown and use this button as a child to trigger it.

Reply:
Agreed! it is fixed in my latest commit. Thank you and I'm open for more feedback if needed